Makefile

```diff
@@ -729,6 +737,8 @@ generate_e2e_test_yaml: $(HELM) $(HOME)/.helm istio-init.yaml
 		--set global.mtls.enabled=true \
 		--set global.controlPlaneSecurityEnabled=true \
 		--set global.proxy.enableCoreDump=${ENABLE_COREDUMP} \
+		--set gateways.istio-egressgateway.enabled=true \
+		--set global.outboundTrafficPolicy.mode=REGISTRY_ONLY \
```

Comment:
Hi @louiscryan could you please add these two new flags into istio-auth-sds.yaml at line 779 below?